easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -62,8 +62,12 @@ def configure_step(self, srcdir=None, builddir=None):
             self.log.nosupport("CMakeMake.configure_step: named argument 'builddir' (should be 'srcdir')", "2.0")
 
         # Set the search paths for CMake
-        include_paths = os.pathsep.join(self.toolchain.get_variable("CPPFLAGS", list))
-        library_paths = os.pathsep.join(self.toolchain.get_variable("LDFLAGS", list))
+        tc_ipaths = self.toolchain.get_variable("CPPFLAGS", list)
+        tc_lpaths = self.toolchain.get_variable("LDFLAGS", list)
+        cpaths = os.getenv('CPATH', '').split(os.pathsep)
+        lpaths = os.getenv('LD_LIBRARY_PATH', '').split(os.pathsep)
+        include_paths = os.pathsep.join(tc_ipaths + cpaths)
+        library_paths = os.pathsep.join(tc_lpaths + lpaths)
```

Comment:
Should we avoid duplicates here? You can use `nub` (which you can import from `vsc.utils.missing`):

```python
include_paths = os.pathsep.join(nub(tc_ipaths + cpaths))
library_paths = os.pathsep.join(nub(tc_lpaths + lpaths))
```